modules/ROOT/pages/Development/Modeling/PaniniProjection.adoc

```diff
@@ -0,0 +1,95 @@
+|✔️ Panini Enabled |❌ Panini Disabled
+
+| image::Development/3DModeling/Panini/Correct_70.png[title=Enabled (70° FOV)]
+| image::Development/3DModeling/Panini/Incorrect_70.png[title=Disabled (70° FOV)]
+
+| image::Development/3DModeling/Panini/Correct_90.png[title=Enabled (90° FOV)]
+| image::Development/3DModeling/Panini/Incorrect_90.png[title=Disabled (90° FOV)]
+
+| image::Development/3DModeling/Panini/Correct_120.png[title=Enabled (120° FOV)]
+| image::Development/3DModeling/Panini/Incorrect_120.png[title=Disabled (120° FOV)]
+
+| image::Development/3DModeling/Panini/Panini_on.png[title=Modded equipment Enabled (90° FOV)]
+| image::Development/3DModeling/Panini/Panini_off.png[title=Modded equipment Disabled (90° FOV)]
+
+|===
+
+== Reusing Base Game Materials
+
+A number of base game materials already have Panini Projection support built in.
+It can be enabled from material instances via their material parameters, usually called something like "UsePaniniProjection".
+
+Examples of base game materials that support Panini Projection include equipment and trinket materials, as well as MM_FactoryBaked.
+For example, trinkets can use the `Asset_Master` material with the "UsePaniniProjection" option enabled.
+This is the approach taken by Example Mod's Example Trinket, Desc_Trinket_TvOfKnowledge.
+
+== Enabling Panini Projection on Custom Materials
+
+Panini can be integrated into custom materials in two main ways, depending on whether your material uses Material Attributes.
+
+[id="MF_ApplyPaniniProjection"]
+=== 1. Using `MF_ApplyPaniniProjection` (Material Attributes workflow)
+
+. In your Material settings, enable *Use Material Attributes*.
+. Use `Make Material Attributes` to gather outputs.+
+. Feed the result into `Apply Panini Projection`.
+. Add a *Static Switch* to toggle Panini on/off.
+. Connect the `Apply Panini Projection` to the *Static Switch True* input.
+. Connect the result of `Make Material Attributes` to the *Static Switch False* input.
+. Connect the result to the Material's *Material Attributes* output.
+
+Use Material Instances to enable toggling the effect per-instance without recompiling the shader.
+This is particularly useful because the models of other players, and 3rd person view models) should not have the effect applied.
+
+image::Development/3DModeling/Panini/Apply_Panini.png[Apply Panini, title=Apply Panini]
+
+[id="MF_PaniniProjection"]
+=== 2. Using `Panini Projection` directly (non-attributes workflow)
+
+. Add a new `MF_PaniniProjection` node to the material graph.
+. Connect the parameters: *d* (distance), *s* (squeeze), *Screen Space Scale*, and *WPO*.
+. Add a *Static Switch* to toggle Panini on/off.
+. Feed the result of the Panini Projection into the *Static Switch True* input.
+. Plug a `Texture Coordinate` Node into the *Static Switch False* input.
+. Plug the result into the respective `Texture Sample` *UV* input.
+. Route the output to your material logic (e.g., Emissive/Color/Roughness, CustomUV).
```

Comment:
Now just do all the other things you were going to do with BaseColor/Emissive/Color/Roughness etc.